OxfordBodleian/Ullendorff/BDLeu10.xml

```diff
@@ -34,59 +37,162 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
-                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r1" to="1r67"/>
+                            <title type="complete" ref="LIT6811Protect"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">በስመ<supplied reason="omitted">፡</supplied> አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ጸሎት፡ በእንተ፡ ማ</hi>ዕሠረ፡
+                                አጋንንት፡ ወአልቦ፡ ስም፡ ዘየሐዩ፡ በታሕተ፡ ሰማይ፡ ወምድር፡ ዘእንበለ፡ ስሙ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ጋኔን፡ ዘትሬኢ፡ በዓይነ፡ ጠባየእ፡ 
+                                እሳታዊ፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                እልመንክር፡ ጋኔን፡ ዘትሬኢ፡ በዓይነ፡ ጠባይዕ፡ እሳታዊ፡ ሹሐል፡ እልመክኑን፡ በዝ፡ አስማተ፡ ስምከ፡ ከመ፡ ኢትቅረቡ፡ ኀበ፡ ነፍሳ፡
+                                ወሥጋሃ፡ ለዓመትከ፡ ወለተ፡ <hi rend="rubric">ሰንበት፡ እንጐቻ።</hi>
+                            </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="1r68" to="1r120"/>
+                            <title type="complete" ref="LIT6853Prayer"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">በስመ<supplied reason="omitted">፡</supplied> አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ፡ ዘአሠሮ፡ ለጸላኢ፡</hi>
+                                በማዕሠረ፡ <del rend="strikethrough">ጸላኢ፡</del> አሌፍ፡ መደንግፅ፡ ማየ፡ ጠፈረ፡ ሰሎሞን፡ ሎፍሐም፡ ሎፍሐም፡ ሎፍሐም፡ መሐፍሎን፡
+                                መሐፍሎን፡ መሐፍሎን፡ ንምሎስ፡ ንምሎስ፡ ንምሎስ፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                ጋኔን፡ ዘትሬኢ፡ በዓይነ፡ ጠባይዕ፡ ማያዊ፡ ተአሠር፡ ይቤ፡ በቃለ፡ እግዚአብሔር፡ ከመ፡ ኢትቅረቡ፡ ኀበ፡ ነፍሳ፡
+                                ወሥጋሃ፡ ለዓመትከ፡ ወለተ፡<hi rend="rubric">ሰንበት፡ እንጐቻ።</hi>
```

Comment:
could you mark up the personal name? Here and below

Reply:
This means that the name will be indexed also in the Ethiopic text, right? Are we ok with that? Sorry for coming back to this question

Reply:
Yes, it will be. As the index will list all occurrences of each name regardless of page breaks, we will need to edit it manually anyway,